sys-libs/llvm-libunwind/llvm-libunwind-12.0.0.ebuild

```diff
@@ -13,7 +13,7 @@ HOMEPAGE="https://github.com/llvm-mirror/libunwind"
 LICENSE="Apache-2.0-with-LLVM-exceptions || ( UoI-NCSA MIT )"
 SLOT="0"
 KEYWORDS="amd64 arm arm64 ~ppc64 ~riscv x86 ~x64-macos"
-IUSE="debug static-libs test"
+IUSE="debug static-libs test compiler-rt"
```

Comment:
I don't think this needs to be a USE flag. If you really need to switch the CMake flag explicitly, it's better to detect what rtlib clang uses. I think one of the packages has code for that that you can reuse.

Reply:
That's the trick...both `clang` and `clang-runtime` have this USE flag, but there's no autodetection. Another option is to switch it ON permanently – but then `compiler-rt` needs to be a hard dependency, and that doesn't cover all possible use cases and seems like against Gentoo spirit to me.

Reply:
I think I got what you mean, looks like a job for `has_use` but I cannot find a proper example

Reply:
This should do